tests/ecmp/test_fgnhg.py

```diff
@@ -27,8 +27,11 @@
 SUPPORTED_TOPO = ['t0']
 SUPPORTED_PLATFORMS = ['mellanox']
 
+MINIGRAPH_PARSING = "OFF"
+
 logger = logging.getLogger(__name__)
 
+
```

Comment:
Just a suggestion: see a lot of diffs with spacing changes, just wondering why that is, if we can limit that then that would be great, but not a big thing even if we commit as-is.

Reply:
Let me look into. This always happens ,  I'm not sure exactly why. I will try to minimize the space delta as much as possible